web-app/src/app/stitch-patterns.html

```diff
@@ -73,14 +73,30 @@
           patternInfos: {
             type: Array,
             value: []
-          }
+          },
+          active: {
+              type: Boolean,
+              reflectToAttribute: true,
+              observer: 'activeChanged'
+            }
         }
       }
 
-      constructor() {
-        super();
-        const json = localStorage.getItem("patternInfos");
-        if (json) this.patternInfos = JSON.parse(json);
+      activeChanged() {
+          if (!this.active) {
```

Comment:
This logic causes pattern load every time user opens this page. We should only clear the data when user gets logged out.